drush/Sites/self.site.yml

```diff
@@ -0,0 +1,14 @@
+# Edit or remove this file as needed.
+# Docs at https://github.com/drush-ops/drush/blob/master/examples/example.site.yml
+
+#live:
```

Comment:
Lets use prod here. It should match with our default policy file

Reply:
drush/**Sites**/self.site.yml should be **sites**?. I looked through the drush code and should not find a instance with ``Sites``.